packages/media/src/types.js

```diff
@@ -68,7 +68,7 @@ ResourcePropTypes.videoResource = PropTypes.shape({
   width: PropTypes.number.isRequired,
   height: PropTypes.number.isRequired,
   poster: PropTypes.string,
-  posterId: PropTypes.number,
+  posterId: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Getting rid of this noise

<img width="913" alt="Screen Shot 2022-01-18 at 2 56 56 PM" src="https://user-images.githubusercontent.com/10720454/150026899-090680c8-fb49-4583-b146-f567e18571be.png">